src/Abp.Zero.Common/Localization/ApplicationLanguageManager.cs

```diff
@@ -57,6 +57,11 @@ public virtual async Task<IReadOnlyList<ApplicationLanguage>> GetLanguagesAsync(
             return (await GetLanguageDictionaryAsync(tenantId)).Values.ToImmutableList();
         }
 
+        public virtual async Task<IReadOnlyList<ApplicationLanguage>> GetActiveLanguagesAsync(int? tenantId)
+        {
+            return (await GetLanguagesAsync(tenantId)).Where(l => !l.IsDisabled).ToImmutableList();
```

Comment:
The reason why I query all languages from db then use where expression is that there is no `Microsoft.EntityFrameworkCore` reference on that `Abp.Zero.Common` and that's why I could not use ToListAsync. 
And since there are just a few languages on db, no need to spend alot time on that query

Reply:
Can we try injecting `IAsyncQueryableExecutor` then?

Reply:
I forget that exists 😄 
But if I query db with that I need to change alot of places (Because it has cache system). 
We can leave it like that because its a table with 10-20 row, it will not cause a performance issue

But thanks, I'il use it when I need it.👍

Reply:
If I query db with that I need to change alot of places (Because it has cache system). 
We can leave it like that because its a table with 10-20 row, it will not cause a performance issue

But thanks. I forgot it exist, I'il use it when I need it👍